### ISSUE TITLE
KALITE_HOME not working

### ISSUE THREAD

**Issue by 66eli77** [learningequality/ka-lite#4684]
Branch: develop
OS: Mac
when set KALITE_HOME = wherever/you/want/
then run kalite, the ```.kalite``` folder is created in wherever/you/want/
but still in your "home" folder

**Comment by benjaoming** [comment]
And you are using an absolute path, i.e. `KALITE_HOME=/an/absolute/path kalite start` ?

**Comment by aronasorman** [comment]
@benjaoming we used `pwd`, which gave an absolute path.

**Comment by benjaoming** [comment]
As in ``KALITE_HOME=`pwd` kalite start`` ?

**Comment by aronasorman** [comment]
Yeah, similar:

```
KALITE_HOME=`pwd`/.kalite kalite start
```
We saw an example which added the `.kalite` dir.

**Comment by benjaoming** [comment]
Is that the exact command?

**Comment by aronasorman** [comment]
No actually. Here's the exact command:

```
KALITE_HOME=`pwd`/.kalite kalite manage setup
```
We expected a folder with .kalite created in the current directory. The DB also should have been created, as the output of that command indicates that it's creating a fresh db. But `ls` does not show any such folder created.

**Comment by MCGallaspy** [comment]
Could it be that spawned subprocesses do not share the given value for KALITE_HOME?

**Comment by benjaoming** [comment]
#worksforme 

On 0.16.x (using the prepopulated db fix PR #4745 )

Notice the last line:

```
KALITE_HOME=".lala" kalite start
Running 'kalite start' as daemon (system service)

Stand by while the server loads its data...

Error reading secret key file. Generating one now. Error was: [Errno 2] No such file or directory: '.lala/secretkey.txt'
INFO:kalite:Setting up KA Lite; this may take a few minutes; please wait!

                                     
   _   __  ___    _     _ _          
  | | / / / _ \  | |   (_) |         
  | |/ / / /_\ \ | |    _| |_ ___    
  |    \ |  _  | | |   | | __/ _ \   
  | |\  \| | | | | |___| | ||  __/   
  \_| \_/\_| |_/ \_____/_|\__\___|   
                                     
https://learningequality.org/ka-lite/
                                     
         version 0.16.0
                                     
WARNING:root:It's recommended that you install Python version 2.7.9. Your version is: 2.7.6

-------------------------------------------------------------------
WARNING: Database file already exists!
-------------------------------------------------------------------
Upgrading database to KA Lite version 0.16.0
Cleaning *pyc files (if writable) from: /home/test/.virtualenvs/kalite_pip/share/kalite/dist-packages
Cleaning *pyc files (if writable) from: /home/test/.virtualenvs/kalite_pip/share/kalite/python-packages
Copying database file from /home/test/.virtualenvs/kalite_pip/lib/python2.7/site-packages/kalite/database/data.sqlite to .lala/database/data.sqlite
```

This also works with `pwd` and absolute paths. Closing this, please investigate on the local deployment where you can recreate it :)
